README.md

```diff
@@ -0,0 +1,34 @@
+## Screens
+
+### Development
+
+#### Prerequisites
+- [Docker](https://www.docker.com/docker-toolbox)
+- [Heroku Toolbelt](https://toolbelt.heroku.com/)
+- Heroku Docker plugin -- `heroku plugins:install heroku-docker`
+
+If running OS X or Windows, [follow these steps for creating a virtual machine for Docker](#creating-a-virtual-machine-for-docker) -- This is a section in this readme file.
+
+#### Setting up development environment
+- Clone the repository -- `git clone git@github.com:ftlabs/screens.git`
+- Change in repository directory -- `cd screens`
+- Build a Docker image -- `docker build .`
+- Spin up the web process in a container -- `docker-compose up web`
```

Comment:
Mention that this is a service in docker-compose.yml. I thought 'web' was a machine name.